includes/autoloader.php

```diff
@@ -129,6 +157,11 @@ private static function init_classes_map() {
 			'WordPress_Widgets_Manager' => 'includes/managers/wordpress-widgets.php',
 		];
 
+		// Who ever uses autoloader, should load 'Controls_Manager' before to support it functionality.
```

Comment:
Extract a base autoloader with an empty `init_classes_map`, extend it from the core and move current `init_classes_map` to the new extended class.

Reply:
Handled.